### PR TITLE
Add distro

### PR DESCRIPTION
**Pull request by windelbouwmanbosch**
I added this for all platforms, but the program only works within windows. Does this make sense?

### CODE REVIEW

File: recipes/distro/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  version: '{{ version }}'
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+  entry_points:  # [linux]
```
Comment (synapticarbors) [inline recipes/distro/meta.yaml]: You don't want to put a linux selector on `entry_points` or `test`. If this package is linux only, you can add under `build`:

```
build:
   skip: True  # [not linux]
```

File: recipes/distro/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+requirements:
+  host:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:  # [linux]
+  commands:
```
Comment (synapticarbors) [inline recipes/distro/meta.yaml]: Can you add an `imports` test to just import `distro`?

File: recipes/distro/meta.yaml (hunk 1 above)
Comment (synapticarbors) [reply recipes/distro/meta.yaml]: Also if you're just building on linux, you could add ` [skip appveyor]` to the commit message as well as `[skip travis]` for osx

Comment (windelbouwmanbosch) [reply recipes/distro/meta.yaml]: Allright. I think this package is intended for linux only.

**Comment by windelbouwmanbosch** [comment]
@conda-forge-admin, please lint

**Comment by windelbouwmanbosch** [comment]
@synapticarbors I think this package is ok now? What you say?

**Pull request merged**